circleci.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -uexo pipefail
+
+HOST_DMD_VER=2.068.2 # same as in dmd/src/posix.mak
+CURL_USER_AGENT="CirleCI $(curl --version | head -n 1)"
+N=2
+
+case $CIRCLE_NODE_INDEX in
+    0) MODEL=64 ;;
+    1) MODEL=32 ;;
+esac
```

Comment:
> than just the latency of parallel builds on many updated PRs.

@klickverbot Let's just see whether it makes a difference and/or clogs our CircleCi queue. It's just an experiment, so we can always switch back to 1 container ;-)